css/style-desktop.css

```diff
@@ -0,0 +1,378 @@
+/* min-width=768px */
+.main-container {
+  width: 100%;
+  height: 4561px;
+  padding: 15px;
+  padding-top: 0%;
+}
+
+.navbar {
+  width: 1440px;
+  height: 72px;
```

Comment:
- [ ] Good job on implementing the app's responsiveness, but you have followed other procedures by creating CSS files based on their screen sizes, and challenging to style with that range of responsiveness. Because of these difficulties, technology comes in CSS, known as media query. In a mobile-first approach, screen size is designed based on the mobile-first principle, so for the media query, If you are preparing for a large-screen, please the media query for the desktop version comes at the end of your CSS, which was used for the mobile version; behind that, the desktop version overrides the mobile version. Kindly recommend using one CSS file and adding a media query for it to be responsive. For details [refer to this ](https://www.w3schools.com/css/css3_mediaqueries.asp). Kindly fix the issue :wrench: 

Example

- Large mobile screen

```
@media only screen and (min-width: 500px) and (max-width: 767px) {
  .headline-content p {
    max-width: 60%;
  }

```
- Medium size screen

```
@media only screen and (min-width: 768px) and (max-width: 999px) {
  .headline-content h1 {
    font-size: 3.625rem;
  }

```
- Desktop and large screen size

```
@media only screen and (min-width: 1000px) {
   .headline-content h1 {
    font-size: 3.825rem;
  }

```